BraveRewardsUI/Ads/AdsViewController.swift

```diff
@@ -41,9 +41,14 @@ public class AdsViewController: UIViewController {
       $0.leading.greaterThanOrEqualTo(view).inset(8)
       $0.trailing.lessThanOrEqualTo(view).inset(8)
       $0.centerX.equalTo(view)
-      $0.width.equalTo(400).priority(.high)
       $0.top.equalTo(view.safeAreaLayoutGuide.snp.top)
       $0.top.greaterThanOrEqualTo(view).offset(4) // Makes sure in landscape its at least 4px from the top
+      
+      if UIDevice.current.userInterfaceIdiom == .pad {
+        $0.width.equalTo(max(view.bounds.width, view.bounds.height) * 0.40).priority(.high)
```

Comment:
Also, if the user multitasks while the ad is currently visible on screen (possible I guess since it stays up for 30s) -- The width of this isn't going to update because its constrained to a constant

Reply:
@kylehickinson I tested the multi-tasking, it still works. I can resize the split view controller and this add resizes with it because of the less than or equal to constraint.

For actual ads, I can't see why I need to fix it there since the actual ads won't show when the panel is open anyway and won't show when the controller isn't top view controller.. but I've added the change there too anyway.

Reply:
This is what I mean:

![IMG_93650C18A835-1](https://user-images.githubusercontent.com/529104/68894735-e790c480-06f5-11ea-
92eb-8d3086be0550.jpeg)

STR:
1. Start multitasking at onboarding start with another app (I choose App Store), half half
2. Make App Store take up 2/3 (making the size of the ad shrink)
3. Make Brave take entire screen

Ad doesn't resize back to 40% because its a constant, it'll always stay as whatever 40% was when it first got created

Reply:
Fixed.!